torch/_prims/__init__.py

```diff
@@ -583,6 +584,13 @@ def _cbrt_aten(a: torch.Tensor):
     type_promotion=ELEMENTWISE_PRIM_TYPE_PROMOTION_KIND.DEFAULT,
 )
 
+trunc = _make_elementwise_unary_prim(
```

Comment:
Should we model trunc as a prim or a reference?

Reply:
Since `trunc` is similar to `ceil`, `floor`, and `round`, I modeled it as a prim.